spec.bs

```diff
@@ -439,9 +440,14 @@ The <dfn for=Navigator method>runAdAuction(|config|)</dfn> method steps are:
     1. [=Queue a global task=] on [=DOM manipulation task source=], given |global|, to resolve |p|
       with null.
   1. Otherwise:
-    1. [=Queue a global task=] on the [=DOM manipulation task source=], given |global|, to resolve
-      |p| with |winner|'s [=generated bid/ad descriptor=]. TODO: resolve |p| with urn-uuid, instead
-      of a URL.
+    1. Let |urn| and |fencedFrameConfig| be the result of TODO: constructing the relevant
+       {{FencedFrameConfig}} with |winner|'s [=generated bid/ad descriptor=].

```

Comment:
I do not believe we can construct a platform object like this (`fencedFrameConfig`) in parallel. This must be constructed on the main thread with the fenced frame config _struct_ as input.

Reply:
The followup PR modifies an existing config rather than construct it from scratch (https://github.com/WICG/turtledove/pull/616/files#diff-6f5a1d8263b0b0c42e2716ba5750e3652e359532647ac934c1c70086ae3ceddaR498): is that fine?

I don't know how to think about constructing [=fenced frame config=] vs {{FencedFrameConfig}} though.